services/github/src/api.js

```diff
@@ -319,6 +319,36 @@ builder.declare({
         msg.branch = body.ref.split('/').slice(2).join('/');
         break;
 
+      case EVENT_TYPES.ISSUE_COMMENT:
+        // Comments on PRs can trigger tasks, too
+        // For this to work, there should be a `/taskcluster cmd` in the comment
+        // Plus repository should have a `.taskcluster.yml` in default branch with
+        // "allowComments: collaborators" in it
```

Comment:
This might need tweaking if we move the config section to `policy.allowComments`